staging/src/k8s.io/apiserver/pkg/admission/cel/metrics.go

```diff
@@ -98,8 +98,14 @@ func (m *ValidatingAdmissionPolicyMetrics) ObserveDefinition(ctx context.Context
 	m.policyDefinition.WithContext(ctx).WithLabelValues(state, enforcementAction).Inc()
 }
 
-// ObserveCheck observes a policy validation check.
-func (m *ValidatingAdmissionPolicyMetrics) ObserveCheck(ctx context.Context, elapsed time.Duration, policy, binding, expression, enforcementAction, params, state string) {
-	m.policyCheck.WithContext(ctx).WithLabelValues(policy, binding, expression, enforcementAction, params, state).Inc()
-	m.policyLatency.WithContext(ctx).WithLabelValues(policy, binding, expression, enforcementAction, params, state).Observe(elapsed.Seconds())
+// ObserveAdmissionWithError observes a policy validation error that was ignored due to failure policy.
+func (m *ValidatingAdmissionPolicyMetrics) ObserveAdmissionWithError(ctx context.Context, elapsed time.Duration, policy, binding, expression, params, state string) {
+	m.policyCheck.WithContext(ctx).WithLabelValues(policy, binding, expression, "allow", params, state).Inc()
+	m.policyLatency.WithContext(ctx).WithLabelValues(policy, binding, expression, "allow", params, state).Observe(elapsed.Seconds())
```

Comment:
param kind#String() doesn't seem like a value we would make a metric label, if isn't needed if we have the policy name, right? policy name and binding name are sufficient info to figure out the params

expression *really* doesn't seem like a value we would make a metric label... that could be *very* large. is the intent here to understand which specific rule was rejecting the request?

Reply:
For 1.26 I'm supportive of dropping them both. I'd like to get feedback on if per-expression metrics are useful, but we look into that post 1.26.

Reply:
I think if there were a shorter guaranteed way to refer to an expression then that'd be potentially good for debugging (knowing which expression in particular was giving problems), but you're right about the size issue - I've removed the label.

The part about params also makes sense (Cici also pointed this out) - I've removed it as well.